// @atlaskit/theme/dist/cjs/components.js
"use strict";
/*
   This file will become the new index for theme once the codemod is mature enough.
   For now we're keeping the index file to avoid having to do a major change.
   Once the codemod is done and all the AK modules have been codeshifted, we delete index.js and rename this file to index + update all the imports
*/
Object.defineProperty(exports, "__esModule", { value: true });
var getTheme_1 = require("./utils/getTheme");
exports.getTheme = getTheme_1.default;
var themed_1 = require("./utils/themed");
exports.themed = themed_1.default;
var AtlaskitThemeProvider_1 = require("./components/AtlaskitThemeProvider");
exports.AtlaskitThemeProvider = AtlaskitThemeProvider_1.default;
var Appearance_1 = require("./components/Appearance");
exports.Appearance = Appearance_1.default;
var Reset_1 = require("./components/Reset");
exports.ResetTheme = Reset_1.ResetTheme;
exports.Reset = Reset_1.Reset;
var Theme_1 = require("./components/Theme");
exports.default = Theme_1.default;
var hoc_1 = require("./hoc");
exports.withTheme = hoc_1.withTheme;
var createTheme_1 = require("./utils/createTheme");
exports.createTheme = createTheme_1.createTheme;
